### PR TITLE
feat: add incoming dln payments

### LINKED ISSUE #1309: DLN incoming
#1300 
#1341

### PR DESCRIPTION
**Pull request by justinenerio**
## Changes

- adds incoming dln payments

## Related issues

Fixes #1309

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [ ] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

File: packages/espressocash_landing/lib/l10n/device_locale.dart
Comment (ookami-kb) [inline packages/espressocash_landing/lib/l10n/device_locale.dart:1]: Why is it not in common package?

File: packages/espressocash_landing/lib/utils/format_amount.dart
Comment (ookami-kb) [inline packages/espressocash_landing/lib/utils/format_amount.dart:1]: Why is it not in common?

File: packages/espressocash_landing/lib/features/payment_request/widgets/dropdown.dart [hunk 1]
```diff
@@ -0,0 +1,233 @@
+// ignore_for_file: avoid-non-null-assertion
```
Comment (ookami-kb) [inline packages/espressocash_landing/lib/features/payment_request/widgets/dropdown.dart:1]: Don't ignore.

File: packages/espressocash_landing/lib/features/web3/web3_service.dart [hunk 2]
```diff
@@ -0,0 +1,158 @@
+// ignore_for_file: avoid-unnecessary-type-casts, unnecessary_cast, avoid_web_libraries_in_flutter
```
Comment (ookami-kb) [inline packages/espressocash_landing/lib/features/web3/web3_service.dart:1]: Don't ignore (except for `avoid_web_libraries_in_flutter`).

**Review by ookami-kb — CHANGES_REQUESTED** [review]

File: packages/espressocash_landing/lib/l10n/device_locale.dart
Comment (justinenerio) [reply packages/espressocash_landing/lib/l10n/device_locale.dart:1]: resolved in pr: https://github.com/espresso-cash/espresso-cash-public/pull/1358

**Pull request closed without merging**